### ISSUE TITLE
Adding Documentation for your website [ Enhancement ]

### ISSUE THREAD

**Issue by CodeWizarz** [upscayl/upscayl.github.io#2]
# Adding documentation will be quite helpful and and the webpage lacks quite some featues just take these as advice :)

**Comment by NayamAmarshe** [comment]
Yeah that's on my todo. I just don't get a lot of time these days. Working 14-15 hours a day, it's hard to find time to spend on the project.
